src/routes/settings/Encrypt.tsx

```diff
@@ -31,19 +31,24 @@ export default function Encrypt() {
     const [error, setError] = createSignal<Error>();
     const [loading, setLoading] = createSignal(false);
 
-    const [_encryptPasswordForm, { Form, Field }] =
+    const [encryptPasswordForm, { Form, Field }] =
         createForm<EncryptPasswordForm>({
             initialValues: {
                 existingPassword: "",
                 password: "",
                 confirmPassword: ""
             },
             validate: (values) => {
+                setError(undefined);
```

Comment:
makes more sense to put this in `handleFormSubmit` because that's where the setter happens

Reply:
Just tried this, but I remembered why its here, we need the error to reset on any change to the form and not only on submit since an error prevents the user from submitting.

Reply:
Due to the `encryptButtonDisabled` the encrypt button is unclickable until the user has made a change to any of the forms

Reply:
ahh k makes sense